src/main/java/eu/europa/ted/efx/model/templates/ContentBlockStack.java

```diff
@@ -12,27 +12,37 @@ public class ContentBlockStack extends Stack<ContentBlock> {
    * will return to the top of the stack again.
    * 
    * @param number the outline number of the child block.
-   * @param content the content of the child block.
-   * @param context the context of the child block.
-   * @param variables the variables of the child block.
+ * @param context the context of the child block.
+ * @param variables the variables of the child block.
```

Comment:
conditionals is missing in the Java docs params

Reply:
done